Dockerfile

```diff
@@ -1,4 +1,12 @@
-FROM node:10.11
+FROM node:10-alpine
+
+# Use dockerize to force indexer to wait for the broker to be ready
```

Comment:
Add `dockerize` to make docker-compose services wait on each other as appropriate.